src/DocumentFormat.OpenXml.Framework/OpenXmlPartWriter.cs

```diff
@@ -56,6 +83,37 @@ public OpenXmlPartWriter(OpenXmlPart openXmlPart, Encoding encoding)
             _xmlWriter = XmlWriter.Create(partStream, settings);
         }
 
+#if FEATURE_ASYNC_SAX_XML
+        /// <summary>
+        /// Initializes a new instance of the OpenXmlPartWriter.
+        /// </summary>
+        /// <param name="openXmlPart">The OpenXmlPart to be written to.</param>
+        /// <param name="encoding">The encoding for the XML stream.</param>
+        /// <param name="useAsync">Whether the writer should be initialized using async</param>
+        public OpenXmlPartWriter(OpenXmlPart openXmlPart, Encoding encoding, bool useAsync)
```

Comment:
I'm tempted to create a new type like `OpenXmlPartWriterSettings` that would expose `Encoding` and `UseAsync` and potentially other properties. i.e. there have been asks for "sanitizing methods" and I think a general settings object here would make that easier to plumb through

Reply:
Added an OpenXmlPartWriterSettings class